### PR TITLE
Updating Maintainers list to be accurate and using the TOC format

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
@ryjones -- what do you think of this page format?  The data in the first table (Roles) is all in the Settings tab, but regular contributors can't see that.  Can they see the Hyperledger Teams pages that I linked into the page?

**Comment by ryjones** [comment]
@swcurran I like it! @tkuhrt ?

**Comment by tkuhrt** [comment]
I like it, generally. The only thing that I would request change is the the contact information being filled in for the maintainers so that we can reach them if needed.

**Comment by swcurran** [comment]
When Ry tagged you on this @tkuhrt , I knew you were going to say that :-).  It's just a pain to track that down.

I'll get one...how about that?  :-)

**Comment by tkuhrt** [comment]
I believe the original intent was to have at least one way of communicating with the maintainer if we needed to contact them, so at least one should be okay @swcurran

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,70 +1,122 @@
...
+| dbluhm          | Daniel Bluhm     | Admin      |      |            | daniel@indicio.tech      | Indicio PBC         |
+| dh1128          | Daniel Hardman   | Admin      |      |            | daniel.hardman@gmail.com | Provident           |
+| shaangill025    | Shaanjot Gill    | Maintainer |      |            | gill.shaanjots@gmail.com | BC Gov              |
+| swcurran        | Stephen Curran   | Admin      |      |            | swcurran@cloudcompass.ca | BC Gov              |
+| TelegramSam     | Sam Curren       | Maintainer |      |            | telegramsam@gmail.com    | Indicio PBC         |
+| TimoGlastra     | Timo Glastra     | Admin      |      |            | timo@animo.id            | Animo Solutions     |
+| WadeBarnes      | Wade Barnes      | Admin      |      |            | wade@neoterictech.ca     | BC Gov              |
+
+Daniel Bluhm <>, Daniel Hardman <daniel.hardman@gmail.com>, Andrew Whitehead <cywolf@gmail.com>, Timo Glastra <timo@animo.id>, Shaanjot Gill <gill.shaanjots@gmail.com>, Sam Curren <telegramsam@gmail.com>, Wade Barnes <wade@neoterictech.ca>
```
Comment (dbluhm) [inline MAINTAINERS.md:38]: Was this line intentional?

**Comment by swcurran** [comment]
Sorry about that — no.  Removed.

**Review by dbluhm — APPROVED** [review]

**Pull request merged**